tests/behaviour/steps/qbwriter.py

```diff
@@ -891,3 +893,13 @@ def assert_uri_style_for_uri(uri_style: URIStyle, uri: str, node):
         assert path.endswith(".csv") or path.endswith(
             ".json"
         ), f"expected {node} to end with .csv or .json"
+
+@then(u'the RDF should contain version specific triples')
+def step_impl(context):
+
+    context.version_triples = context.text + "\n" + f"prov:used <{get_csvcubed_version_uri()}> ."
```

Comment:
We should use `os.linesep` instead of `\n` so that new lines will be recognised in Windows.